src/components/MetricCard/index.tsx

```diff
@@ -48,7 +55,9 @@ const MetricCard = ({
       </LimitsContainer>
       <CurrentValueContainer>
         <View style={{ flex: 1, justifyContent: 'center' }}>
-          <PresetValue>{setValue.toFixed(0)}</PresetValue>
+          <PresetValue>
+            {setValueText ? setValueText : setValue.toFixed(0)}
```

Comment:
this 
{setValueText ? setValueText : setValue.toFixed(0)}

can be this 

{ setValueText || setValue.toFixed(0) }